frontend/src/metabase-lib/compat/compat.ts

```diff
@@ -0,0 +1,96 @@
+import * as ML from "metabase-lib";
+import type { FieldFilter } from "metabase-types/api";
+import type StructuredQuery from "metabase-lib/queries/StructuredQuery";
+import LegacyFilter from "metabase-lib/queries/structured/Filter";
+
+export const getOperatorsMap = ({
+  query,
+  column,
+  stageIndex = -1,
+}: {
+  query?: ML.Query;
+  column?: ML.ColumnWithOperators;
+  stageIndex?: number;
+}) => {
```

Comment:
WDYT about making the signature consistent with the rest of MLv2 functions? Most functions usually expect a `query` and a `stageIndex` as first two arguments